index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>ERNEST Portfolio Website</title>
+</head>
+<body>
+    <!-- defining the header and wraping it in divs and will contain the logo-->
+    <div class="main_container">
+        <div class="inside_container">
+            <header class="main_header">
+              <nav class="navbar">
+                <a href="#" class="my-portfolio-logo">
+                    <img src="./assets/MyLogo.png" alt="My-logo"/>
+                </a>
+                <a><img src="./assets/menu.png" alt="menu-icon"/></a>
+              </nav>
+            </header>
+    <!-- Defining the main section for the description of the developer -->
+            <section class="main_section">
+                <div class="container">
+                  <h1 class="h1">I am Ernest Lasten. Glad to see you!</h1>
+                    <p class="main_content">
+                        I’m a software developer! I can help you build a product ,
+                        feature or website Look through some of my work and experience!
+                        If you like what you see and have a project you need coded, 
+                        don’t hestiate to contact me.
+                    </p>
+                    <div class="connect_links">
+                        <strong class="connect_text">LET'S CONNECT</strong>
```

Comment:
- [ ] This is a link, please consider using ```<a>``` tag instead of ```<strong>``` so the link has a call to action